src/helpers/slack.ts

```diff
@@ -0,0 +1,109 @@
+const { Reaction, ErrorCode } = require('@slack/web-api');
+import SLACK_MESSAGE_BLOCKS from './files/welcomeMessageBlocks.json';
+import { SlackEvent, SlackWebClient } from './types';
+const fs = require('fs');
+
+
+function getEmojisToReactWith(text: string): Array<string> {
+    const emojisToReactWith: Array<string> = [];
+
+    let rawdata = fs.readFileSync('./helpers/files/emojis.json');
+    let emojis: Record<string, Array<string>> = JSON.parse(rawdata);
+
+    for (const [emoji, keywords] of Object.entries(emojis)) {
+        for (const keyword of keywords) {
+            if (text.includes(keyword)) {
+                emojisToReactWith.push(emoji)
+            }
+        }
+    }
+
+    console.log(`Emojis to react with: ${emojisToReactWith}`)
+    return emojisToReactWith
```

Comment:
```suggestion
    return emojisToReactWith;
```

Time to add something like prettier to autoformat these kinds of mundane things 😄

Reply:
I did add prettier formatting 🙈